code/game/gamemodes/cult/items/armor.dm

```diff
@@ -19,8 +19,8 @@
 	icon_state = "cult_helmet"
 	armor = list(
 		melee = ARMOR_MELEE_VERY_HIGH,
-		bullet = ARMOR_BALLISTIC_SMALL,
-		laser = ARMOR_LASER_MAJOR,
+		bullet = ARMOR_BALLISTIC_CARBINE,
+		laser = ARMOR_LASER_RIFLE,
```

Comment:
Oversight here?

Reply:
that armor is the upgraded variant. It requires the cult to create an artificer. It's always had a value one step above the default summoned robes.

Reply:
nevermind got why you were confused. Fixed the chest piece to match.